src/polyswarmd/__init__.py

```diff
@@ -79,7 +77,7 @@ def check_auth_response(api_response):
     try:
         return api_response.json()
     except ValueError:
-        logger.exception('Invalid response from API key management service, received: %s', r.content)
+        logger.exception('Invalid response from API key management service, received: %s', api_response.encode())
```

Comment:
I always think we should let exceptions bubble and treat them in the correct exception-treating layer or interested function. This pattern forces the caller to check for the return value, which is not pythonic in general. If this requires a larger change, however, I would not try to accomplish this right now.